cluster/images/hyperkube/Makefile

```diff
@@ -15,44 +15,14 @@
 # Build the hyperkube image.
 #
 # Usage:
-#   [ARCH=amd64] [REGISTRY="gcr.io/google_containers"] make (build|push) VERSION={some_released_version_of_kubernetes}
+#   [ARCH=amd64] [REGISTRY="gcr.io/google-containers"] make (build|push) VERSION={some_released_version_of_kubernetes}
 
-REGISTRY?=gcr.io/google_containers
+REGISTRY?=gcr.io/google-containers
```

Comment:
in older versions of docker (pre-1.5 I think) dashes weren't allowed in the registry, so we had to use `google_containers`, which was an alias of the real registry, which comes from the GCP project name, `google-containers`.

I'm slowly trying to standardize everything on `google-containers`. In most places they are interchangeable, though sideloading might cause issues if we are inconsistent, and some `gcloud` commands don't handle the underscored alias properly.